circle.yml

```diff
@@ -1,3 +1,10 @@
 machine:
   node:
-    version: 5.0.0
\ No newline at end of file
+    version: 5.0.0
+deployment:
+  npm:
+    branch: master
+    commands:
+      # login using environment variables
+      - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login
+      - npm run 2npm
```

Comment:
@donnapep this module have some hand stuff with nice error messages. I looked up on the web and it seems to be used a lot. cheers